website/lib/markdown/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable node/no-extraneous-require */
+/* eslint-env node */
+
+'use strict';
+const path = require('path');
+const resolve = require('resolve');
+
+const MarkdownProcessIncludeDirectives = require('./markdown-process-includes');
+const MarkdownToJson = require('./markdown-to-jsonapi');
+const TableOfContents = require('./table-of-contents');
+const MergeTrees = require('broccoli-merge-trees');
+const Funnel = require('broccoli-funnel');
+
+module.exports = {
+  name: 'markdown',
```

Comment:
I didn't spend much time thinking of what to call this so if there is a better suggestion I'm happy to change it

Reply:
in any case we can change it later, so I would not worry too much about it